docs-site/antora-playbook.yml

```
@@ -17,9 +17,9 @@ content:
     - url: ../Vulkan-Samples
       branches: antora
```

```suggestion
      branches: main
```

I think this will fix the missing Samples module problem - overlooked when I switched the repo in CI